config/application.yml.example

```diff
@@ -0,0 +1,4 @@
+MAPS_API_KEY: "G00GL3-M4PS-4P1-K3Y"
+EXTERNAL_ORIGIN_URL_ALLOWED: "www.example.com"
```

Comment:
I'd use the one that would be used in locally.
Like you did previously `locahost:3001`